core/src/util.jl

```diff
@@ -295,27 +296,31 @@ function reduction_factor(x::T, threshold::Real)::T where {T <: Real}
     end
 end
 
-function get_low_storage_factor(
-    current_low_storage_factor::Vector{T},
-    id::NodeID,
-)::T where {T}
-    return id.type == NodeType.Basin ? current_low_storage_factor[id.idx] : one(T)
+# SparseConnectivityTracer overloads
+
```

Comment:
These aren't overloads right? Perhaps this should replace `# Custom overloads` on line 771?
```suggestion
```

https://github.com/Deltares/Ribasim/blob/78e049752c215c5ee323a352ece80d4079ec4dfc/core/src/util.jl#L771